lib/scenic/schema_dumper.rb

```diff
@@ -30,7 +30,11 @@ def dumpable_views_in_database
     unless ActiveRecord::SchemaDumper.private_instance_methods(false).include?(:ignored?)
       # This method will be present in Rails 4.2.0 and can be removed then.
       def ignored?(table_name)
-        ["schema_migrations", ignore_tables].flatten.any? do |ignored|
+        [
+          "schema_migrations",
+          "pg_stat_statements_info",
```

Comment:
I'm not sure we should modify this method. It's a direct copy of the method that existed pre Rails 4.2.0. We're covered by the change in the query for any version of Rails.